ui/components/MesheryMeshInterface/PatternService/RJSF.js

```diff
@@ -144,11 +118,17 @@ function RJSF(props) {
     return () => clearTimeout(timer);
   }, [data]);
 
+  React.useEffect(() => {
```

Comment:
Memoization happening here. Seems like a great improvement. When/where were we performing this rendering previously, if not in UseEffect?

Reply:
on every render, i.e, on every change you were doing to the text box or any form, writing something inside it. The process was running everytime. But since the functional is dependent on schema, it only gets called once schema is changed and the result is saved in a state variable.